source/_integrations/rotarex.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: "Rotarex DIMES SRG"
+description: "Instructions on how to integrate your Rotarex gas tanks into Home Assistant."
+documentation: https://rotarexsrg.com/category/digital-measurement-system
+ha_release: 2026.1
+ha_domain: rotarex
+ha_iot_class: cloud_polling
+ha_codeowners:
+  - '@Rotarex'
+---
+
+The Rotarex DIMES SRG {% term integration %} allows you to monitor the level, battery, and last sync time of your Rotarex DIMES SRG Camping gas tanks in Home Assistant.
```

Comment:
```suggestion
The **Rotarex DIMES SRG** {% term integration %} allows you to monitor your Rotarex DIMES SRG Camping gas tanks in Home Assistant.
```
do you want to keep the intro more generic? so that you don't have to change it every time you add new functionality in future?